src/utils/passwords.ts

```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import bcrypt from 'bcrypt'
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+export const hashPassword = async (password: string) => {
+  const slatRounds = await bcrypt.genSalt(
+    parseInt(process.env.SLAT_ROUND, 10) || 10,
```

Comment:
```suggestion
    SLAT_ROUND,
```
you can export from `src/config` and use it and import it here